<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>251</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="hw1">
         <property name="text">
          <string>Hw1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hw2">
         <property name="text">
          <string>Hw2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hw3">
         <property name="text">
          <string>Hw3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hw4">
         <property name="text">
          <string>Hw4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hw5">
         <property name="text">
          <string>Hw5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hw6">
         <property name="text">
          <string>Hw6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hw7">
         <property name="text">
          <string>Hw7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hw8">
         <property name="text">
          <string>Hw8</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSlider" name="hw1_s">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw2_s">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw3_s">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw4_s">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw5_s">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw6_s">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw7_s">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hw8_s">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>139</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_course">
       <property name="text">
        <string>Course:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="box_course">
       <item>
        <property name="text">
         <string>PIC 10C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PIC 10B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PIC 10A</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>181</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="mt1">
       <property name="text">
        <string>Midterm 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mt1box"/>
     </item>
     <item>
      <widget class="QSlider" name="mt1slide">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mt2">
       <property name="text">
        <string>Midterm 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mt2box"/>
     </item>
     <item>
      <widget class="QSlider" name="mt2slide">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="final_2">
       <property name="text">
        <string>Final</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="finalbox"/>
     </item>
     <item>
      <widget class="QSlider" name="finalslide">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>181</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QRadioButton" name="sch_a">
       <property name="text">
        <string>Schema A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="sch_b">
       <property name="text">
        <string>Schema B</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="Overall">
         <property name="text">
          <string>Overall Score:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="score_o">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGrade_Calculator">
    <property name="title">
     <string>Grade Calculator</string>
    </property>
   </widget>
   <addaction name="menuGrade_Calculator"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
